### PR TITLE
Overhaul of two parts of getting started section

### PR DESCRIPTION
**Pull request by mooreds**
This fixes up some issues with a few of the getting started pages:

* some wording tweaks for the basic registration tutorial. updated video. Added to correct nav section.
* reworked user registration/login via API tutorial as it wasn't really useful.

### CODE REVIEW

File: site/docs/v1/tech/tutorials/register-user-login-api.adoc [hunk 1]
```diff
@@ -1,26 +1,48 @@
...
+They also include a number of common workflows, including, but not limited to:
+
+include::docs/v1/tech/shared/_list-hosted-login-pages-use-cases.adoc[]
 
-== Login a User
+The reason to use the APIs is to give you full control over the workflow, user experience, and look and feel of your login and registration functionality. Reasons include:
 
-Once you have create a User and registered them for an Application, you can authenticate them by calling the link:/docs/v1/tech/apis/login[Login API].
+* You don't want to use a webview for authentication on a mobile application.
+* You need a custom user registration flow not supported by FusionAuth, such as directing a user to different registration flows based on their email addrss.
```
Comment (jobannon) [inline site/docs/v1/tech/tutorials/register-user-login-api.adoc:46]: spelling

addrss

File: site/docs/v1/tech/tutorials/register-user-login-api.adoc [hunk 2]
```diff
@@ -1,26 +1,48 @@
...
+
+Once you have create a User and registered them for an Application, you can authenticate them by calling the link:/docs/v1/tech/apis/login[Login API]. That will return a JWT and other User information as documented.
+
+A user can also log in via the link:/docs/v1/tech/core-concepts/integration-points/#hosted-login-pages[hosted login pages]. These use the Authorization Code grant and will redirect the user to FusionAuth.
+
+image::tutorials/login-page.png[Example of login form.,width=1200, role=bottom-cropped]
+
+== Should I Use the APIs or The Hosted Pages
+
+In general the hosted login pages are recommended. They are link:/docs/v1/tech/themes/[customizable and localizable with themes] and when used, FusionAuth is the only server side system to see sensitive user information like credentials.
```
Comment (jobannon) [inline site/docs/v1/tech/tutorials/register-user-login-api.adoc:37]: FusionAuth is the only server-side system to see sensitive user information like credentials (in other words, you are delegating authentication completely to FusionAuth).

File: site/docs/v1/tech/shared/_list-hosted-login-pages-use-cases.adoc [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+* Two factor authentication
+* Passwordless authentication
+* Password expired
+* Account lockout by administrative decision or failed attempts
+* Password validation failed
+* Breach password detection
+* Federated log in with IdPs such as Google and Microsoft Active Directory
+* Advanced self service registration forms
+* Linking between IdP accounts and FusionAuth accounts
+* Multi application logout (OAuth front channel logout)
```
[the remark this answers is not in the document]
Comment (mooreds) [reply site/docs/v1/tech/shared/_list-hosted-login-pages-use-cases.adoc:17]: Would rather leave the link off if that's okay.

File: site/docs/v1/tech/tutorials/register-user-login-api.adoc [hunk 4]
```diff
@@ -1,26 +1,48 @@
...
+
+== Log in a User
+
+Once you have created a User and registered them for an Application, you can authenticate them by calling the link:/docs/v1/tech/apis/login[Login API]. That will return a JWT and other User information as documented.
+
+A user can also log in via the link:/docs/v1/tech/core-concepts/integration-points/#hosted-login-pages[hosted login pages]. These use the Authorization Code grant or a SAML flow. FusionAuth owns the user interface for authentication in this scenario.
+
+image::tutorials/login-page.png[Example of login form.,width=1200, role=bottom-cropped]
+
+== Should I Use the APIs or The FusionAuth Hosted Login Pages
```
Comment (jobannon) [inline site/docs/v1/tech/tutorials/register-user-login-api.adoc:35]: > The FusionAuth Hosted Login Pages 

the

File: site/docs/v1/tech/tutorials/register-user-login-api.adoc [hunk 5]
```diff
@@ -1,26 +1,48 @@
...
+
+In general the hosted login pages are recommended. They are link:/docs/v1/tech/themes/[customizable and localizable with themes] and when used, FusionAuth is the only server side system to see sensitive user information like credentials. In other words, you are delegating authentication entirely to FusionAuth.
```
Comment (jobannon) [inline site/docs/v1/tech/tutorials/register-user-login-api.adoc:37]: Is it just authentication?  I think this word works?

Should we say AuthN or AuthZ or just Auth?

**Review by jobannon — APPROVED** [review]
Looks good.  Few smaller suggestions.

**Pull request merged**